crates/cuda_builder/src/lib.rs

```diff
@@ -266,6 +271,19 @@ impl CudaBuilder {
         self
     }
 
+    /// Enable fast math approximations globally (equivalent to NVCC's --use_fast_math).
+    /// This implies ftz=true, prec-div=false, prec-sqrt=false, and fmad=true.
+    pub fn fast_math(mut self, fast_math: bool) -> Self {
+        self.fast_math = fast_math;
+        if fast_math {
+            self.ftz = true;
+            self.fast_sqrt = true;
+            self.fast_div = true;
+            self.fma_contraction = true;
```

Comment:
This is a redundant representation. You're storing `fast_math` and also the individual flags. It should do one or the other.